src/database/config/build.js

```diff
@@ -0,0 +1,19 @@
+const { readFileSync } = require('fs');
+const { join } = require('path');
+require('env2')('./config.env');
+
+const connection = require('./connection');
```

Comment:
`env2` is is called within the `connection` file, what happens when you require a file is that it will be read(and hence the `env2` call will be seen) and whatever you are assigning to the `module.exports` object will be returns, and hence you don't need to call the `env2` here